test/model/mapping/coercer_test.rb

```diff
@@ -16,22 +17,41 @@
       coercer.to_record(entity).must_equal(name: 'Tyrion Lannister')
     end
 
+    describe 'new record' do
+      # Bug:
+      # https://github.com/lotus/model/issues/155
+      it 'ignores unset values' do
+        entity = User.new(name: 'Daenerys Targaryen')
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
+      end
+
+      it 'forces nil values' do
+        entity = User.new(name: 'Daenerys Targaryen', age: nil)
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
```

Comment:
@AlfonsoUceda @joneslee85 Sorry but @luishurtado is right.

It may make not sense to send the value for databases with a schema, where the value will remain `nil`/`NULL`.

The issue arises for schema-less databases, when a developer sends the key, but it gets ignored and the structure of the collection isn't altered according their needs. Think of Redis or Mongo in this case.

@AlfonsoUceda Can you please take care of this? Thank you. https://github.com/lotus/model/issues/163